vk_video_decoder/libs/NvVideoParser/include/VulkanH264Decoder.h

```diff
@@ -809,7 +818,7 @@ class VulkanH264Decoder: public VulkanVideoDecoder
     bool dpb_sequence_start(slice_header_s *slh);
     void dpb_picture_start(pic_parameter_set_s *pps, slice_header_s *slh);
     void dpb_picture_end();
-    VkPicIf *alloc_picture();
+    VkPicIf *alloc_picture(uint32_t width, uint32_t height);
```

Comment:
I don't think we need the dimensions with alloc_picture() here. There are no pictures allocated, but rather just a picture allocated from a image pool with a fixed image size.